### ISSUE TITLE
element IDs are not unique

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3909]
example page is https://www.iastatedigitalpress.com/mmb/article/id/13894/

tables and <a> tags have duplicate element IDs, possibly more than those.

**Comment by ajrbyers** [comment]
This issue is caused by the table being duplicated for the View Larger modal. The new table should be assigned a new ID.

**Comment by StephDriver** [comment]
#4055 added to consider use of tables in general, which will then drill down into A11y improvements for each usecase.
